### DISCUSSION TITLE
Customer Fields Manager not showing fields on registraion page

### DISCUSSION THREAD

**Discussion by wdmarketLV** [breezefront/community#85] [1. Help]
Hello, I did create custom fields, I mean custom. 
1. They are not showing in the front end. 
2. Also I would like to add radio buttons there, is that possible? 
3. Can you send how can I create them in two steps? 

![image](https://user-images.githubusercontent.com/79901550/229869767-f17de789-78f9-4c2c-b09e-dfe0357a9b92.png)
![image](https://user-images.githubusercontent.com/79901550/229869835-ef00ddaf-d5a5-4014-ac59-af6ca3a0f7bb.png)

**Comment by wdmarketLV** [comment]
Hello, any updates here?

**Comment by vovayatsyuk** [comment]
Hello, sorry for the delay.

1. This module is not yet integrated with Breeze - we forgot about it :( I will start integration next week.
2. No it's not possible yet, but I think we can add this feature.
3. Do you mean some kind of multistep wizard form? No, it's not possible.

**Comment by vovayatsyuk** [comment]
Please update swissup modules to their latest versions to get Breeze integrations:

```
composer update "swissup/*" -W
```

**Comment by wdmarketLV** [comment]
Hello, thank you very much. 
Now is working in the registration form. 
But not in checkout and didn't check in the admin panel. 

Also is it possible to create radio buttons in customer fields as it is in checkout fields? 

Best Regards

**Reply by vovayatsyuk** [reply]
That’s correct. Customer field manager fields are shown at customer forms only.

Radio buttons: I will add this request to our repository.

**Reply by wdmarketLV** [reply]
OK. 
Thanks, how does it look with radio buttons? 
Regards

**Comment by wdmarketLV** [comment]
Ok, thanks for info. 

That’s correct. Customer field manager fields are shown at customer forms only. 
Is there way to show them in checkout or create ralitions on checkout also? 
Regards

**Reply by vovayatsyuk** [reply]
No, it's not possible. Address or checkout fields are the only fields that will appear at checkout.

**Reply by wdmarketLV** [reply]
Ok
